api/src/wfl/util.clj

```diff
@@ -432,3 +434,81 @@
    (poll task seconds 3))
   ([task]
    (poll task 1)))
+
+(defn columns-rows->tsv
```

Comment:
Of the new code I've added here, this is the only method I feel confident should live in `util`.